docs/source/tutorial/tutorial.parallelism.rst

```diff
@@ -65,27 +65,29 @@ For example, you create the following file called ``run_learner.py``:
 
     from mpi4py.futures import MPIPoolExecutor
 
-    learner = adaptive.Learner1D(f, bounds=(-1, 1))
+    if __name__ == "__main__":  # ← use this, see warning @ https://mpi4py.readthedocs.io/en/stable/mpi4py.futures.html#mpipoolexecutor
```

Comment:
Sorry again, do you mind moving this to a separate line? This is too long otherwise.

Other than that LGTM